app/models/ability.rb

```diff
@@ -8,7 +8,10 @@ def initialize(user)
     if user.administrator?
       can :manage, :all
       can :create, Course
-      can :refresh, Course
+      cannot :refresh, Course
```

Comment:
I think admins should be allowed to refresh the course just in case of we need to do some sort of maintenance etc.